tools/scenario-player/requirements.txt

```diff
@@ -0,0 +1,3 @@
+blessings==1.7
+raiden==0.9.0
```

Comment:
Had to remove this when installing in the same venv as the raiden one. Maybe if we don't pin the Raiden version pip would detect it was already installed and skip this requirement? with the added benefit that we test the latest available version of raiden on PyPI if you don't have it already installed.

Reply:
Yes, that should work.